### ISSUE TITLE
Incompatability with `ispell` on lines containing no words

### ISSUE THREAD

**Issue by adamsmd** [rrthomas/enchant#251]
Passing a line that contains no words to `enchant` in pipe mode (i.e., option `-a`) produces a different result from `ispell`.

Specifically `ispell` produces a blank line to signal that all words in a line have been handled.  (Quoting from the `ispell` man page: "Each sentence of text input is terminated with an additional blank line, indicating that ispell has completed processing the input line.")

However, `enchant` doesn't do that when there are no words on a line.  

As an example suppose the input line is `...`, then `enchant` produces the following with no trailing blank line.

```
$ echo '...' | enchant -a
@(#) International Ispell Version 3.1.20 (but really Enchant 1.6.0)
$
```

This is different than `ispell`, `aspell` and `hunspell`, which produce a blank line:

```
$ echo '...' | ispell -a
@(#) International Ispell Version 3.4.00 8 Feb 2015

$ echo '...' | aspell -a
@(#) International Ispell Version 3.1.20 (but really Aspell 0.60.8)

$ echo '...' | hunspell -a
@(#) International Ispell Version 3.2.06 (but really Hunspell 1.7.0)

$
```

**Comment by rrthomas** [comment]
This has been fixed in the current version of enchant. Not sure how long ago.

**Comment by rrthomas** [comment]
I did a bit of digging, and this bug seems to have been fixed in v1.6.1, which was released about 3½ years ago. Please try to test bugs against a more current version in future! (I appreciate that some downstreams are way out of date, but all I can do there is help them package newer versions—I don't have the bandwidth to support users of old versions directly. Feel free to report bugs such as this to your distro's bug tracker, it might help get enchant updated!)
